### PR TITLE
ADDED SLACK LINK

### PR DESCRIPTION
**Pull request by athulmekkoth**
## Purpose

The purpose of this PR is to fix #<issue-number>

### Preview Link

https://pr-{PR_NUMBER}-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/content/static/navbar.html [hunk 1]
```diff
@@ -139,6 +139,16 @@ <h6 class="text-primary mb-md-1">Student Ambassador</h6>
...
+                <li class="nav-item">
+                    <a class="nav-link nav-link-icon" href="https://join.slack.com/t/sefheadquarters/shared_invite/zt-1ds9bjtli-ry~ggwf2kSJ_j5src89GBQ" target="_blank"
+                       data-toggle="tooltip" title="" data-original-title="Join us on Slack">
+                        <i class="fab fa-slack"></i>
+                        <span class="nav-link-inner--text d-lg-none">Slack</span>
+                    </a>
+                </li>
+
+
+
```
Comment (anjula-sack) [inline assets/content/static/navbar.html]: Can you remove these extra lines?

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Thank you! @athulmekkoth 
can you update the preview link in the description as well? 
[https://pr-1213-sef-site.surge.sh/](https://pr-%7Bpr_number%7D-sef-site.surge.sh/)

and link the issue as well

File: assets/content/static/navbar.html (hunk 1 above)
Comment (athulmekkoth) [reply assets/content/static/navbar.html]: sure

**Review by athulmekkoth — COMMENTED** [review]
edited blank space

**Review by anjula-sack — APPROVED** [review]
Thank you! @athulmekkoth

**Pull request merged**